src/icons/fileIcons.ts

```diff
@@ -2420,5 +2420,9 @@ export const fileIcons: FileIcons = {
       name: 'container',
       fileNames: ['.devcontainer/devcontainer.json'],
     },
+    {
+      name: 'bruno',
+      fileExtensions: ['bru'],
```

Comment:
```suggestion
      fileExtensions: ['bru'],
      light: true,
```

Reply:
I don't think this is necessary because this icon will look the same on both dark and light backgrounds

Reply:
I don't think this is necessary because this icon will look the same on both dark and light backgrounds